### PR TITLE
Modal working fine and Testing issue resolved temporarily

### PR DESCRIPTION
**Pull request by horaanchal**

### CODE REVIEW

File: web/src/app/app.module.ts [hunk 1]
```diff
@@ -25,14 +25,13 @@ import { AppNavBarComponent } from './nav-bar/nav-bar.component';
 import { ListComponent } from './reusable-components/list/list.component';
 import { ModalComponent } from './reusable-components/modal/modal.component';
 import { ScheduleInterviewComponent } from './schedule-interview/schedule-interview.component';
-import { JdModalComponent } from './jd-modal/jd-modal.component'
+import { JdModalComponent } from './jd-modal/jd-modal.component';
```
Comment (sonalichawla16) [inline web/src/app/app.module.ts:28]: dont commit pdf file

File: web/src/app/jd-modal/jd-modal.component.ts [hunk 2]
```diff
@@ -105,32 +108,44 @@ export class JdModalComponent implements OnInit {
...
-            setTimeout(()=> { 
-              this._router.navigate(["/edit"]);
-            }, 1000);
-        }
-        else if(res.status == 401){
-          this._router.navigate(['/login']);
-        }
-      });
+      console.log(jdFormObject)
+        this._service.updateJobInfo(jdFormObject,this.jobArray._id).subscribe((res: any) => {
```
Comment (sonalichawla16) [inline web/src/app/jd-modal/jd-modal.component.ts:125]: ???console logs

Comment (horaanchal) [reply web/src/app/jd-modal/jd-modal.component.ts:125]: console log removed in later commits.

File: web/src/app/app.module.ts (hunk 1 above)
Comment (horaanchal) [reply web/src/app/app.module.ts:28]: pdf files deleted

**Comment by horaanchal** [comment]
changes has been made, please merge.

**Pull request merged**